### PR TITLE
New site

### PR DESCRIPTION
**Pull request by skipzero**
cleaned up the merge conflicts I added when oking the last PR. sry.

### CODE REVIEW

**Comment by teone** [comment]
Looks good to me, just a couple notes:
- in `gulpfile.js` instructions it states `Now you will just use $plugin and ...` while it should be `$plug` (nice improvement~)
- in `gulpfile.js` remove the unused requires (`handlebars`, `rename`, `babel`)

What about creating our own `_vars.scss` to override bootstrap defaults?

And last (just to be picky :) can you squash your commits into a single `Added bootstrap` (or similar)? It will make the history clearer.

Well done and thanks!

**Comment by skipzero** [comment]
yeah i can remove those i more left them in case y'all wanted to have me revert the plugin back. ill get all the changes + history done tonight

**Comment by skipzero** [comment]
created a _var.scss file and set it up for import, currently empty but ready to roll. just need to squash my commits and should be ready to roll.

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+6
```
Comment (llkats) [inline .nvmrc]: If `.nvmrc` is being added to `.gitignore`, then please remove this file from git.

File: build.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+'use strict';
```
Comment (llkats) [inline build.js]: This file was removed in 09357c24f7b1cdb. Should it be included back in, or was this a merge artifact?

File: .nvmrc (hunk 1 above)
Comment (skipzero) [inline .nvmrc]: Didn't realise it'd get in, it was just for me. Since I was building the binary for gulp-sass, I wanted it to stay on the correct version for the binary. My apologies.

File: data.json [hunk 3]
```diff
@@ -10,28 +10,21 @@
       },
       "location": "tbd",
       "registerLinks": {
-        "mentor": "https://github.com/nodeschool/sanfrancisco/issues/193",
```
Comment (llkats) [inline data.json]: Looks like this is clobbering edits from f441aff0c05d148.

File: build.js (hunk 2 above)
Comment (skipzero) [inline build.js]: artifact. I'll remove that as well.  Also, while I'm here, should I do a new PR with flattened history? I think I squashed it, but still showed the old. TBH, I've never actually squashed my commit history. I feel silly that I dont know this... :)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,333 @@
+<!DOCTYPE html>
```
Comment (llkats) [inline index.html]: I'm confused about this file - is this one the canonical index.html, or is it `dist/index.html`? We should probably only have one. This file also includes some merge conflict markup that needs to be removed before merge.

Comment (llkats) [inline index.html]: Oh lol on second look, it looks like the old homepage index.html. Definitely don't need it, then.

Comment (skipzero) [inline index.html]: That one I was confused over too. I actually threw out the whole directory all this was in twice in my attempts to squash all history... :/ 
I'll make the above changes, and try squashing and upload now. Sorry about the mess...

**Comment by llkats** [comment]
I also noticed a yarn.lock file -- surely we don't need to introduce yarn into this on top of everything?

**Comment by skipzero** [comment]
nope. I've tried keeping the extras out, and failed miserably...

File: build.js (hunk 2 above)
Comment (llkats) [inline build.js]: @skipzero if you want to squash your commits, it'd be better to do that to a fresh branch and make a new PR, now that you've already pushed it up. I'm personally not going to be a stickler about it, in this case, and will be fine merging multiple commits (I've already done that for my own commits, so no worries). I'd rather work off of this existing PR so the comments/feedback are in place.

**Comment by skipzero** [comment]
I'll get the changes done and let's start the PR over!? 😄

**Comment by llkats** [comment]
@skipzero sounds good. Take your time, I'll have more cycles for review tomorrow!

**Comment by skipzero** [comment]
cool. I'll still get these done and up tonight. I don't want to make more work for anyone. 👍

File: index.html (hunk 4 above)
Comment (teone) [inline index.html]: I think I may forgot this file around when I moved everything in `src`, sorry about that!

**Comment by llkats** [comment]
Friends, I am having all sorts of troubles just getting the build to run in @skipzero's branch. I have tried blowing away my `node_modules` folder and reinstalling, and that fixed some errors, but not all of them. 

To check out @skipzero's changes, I ran:

    git checkout -b skipzero-new-site new-site
    git pull https://github.com/skipzero/sanfrancisco.git new-site

Error output:

```~/Developer/sanfrancisco ( skipzero-new-site* )
(☞ ﾟヮﾟ)☞  npm start

> NodeSchoolSF@2.0.0 start /Users/lydia/Developer/sanfrancisco
> gulp

[11:51:37] Using gulpfile ~/Developer/sanfrancisco/gulpfile.js
[11:51:37] Starting 'hbs:compile'...
[11:51:37] Starting 'scripts'...
[11:51:37] Starting 'assets'...
[11:51:37] Starting 'styles'...
[11:51:37] Starting 'watch'...
[11:51:37] 'watch' errored after 7.81 ms
[11:51:37] TypeError: Cannot read property 'src' of undefined
    at Gulp.gulp.task (/Users/lydia/Developer/sanfrancisco/gulpfile.js:112:30)
    at module.exports (/Users/lydia/Developer/sanfrancisco/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/lydia/Developer/sanfrancisco/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Users/lydia/Developer/sanfrancisco/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/Users/lydia/Developer/sanfrancisco/node_modules/orchestrator/index.js:134:8)
    at /Users/lydia/Developer/sanfrancisco/node_modules/gulp/bin/gulp.js:129:20
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickCallback (internal/process/next_tick.js:104:9)
    at Module.runMain (module.js:607:11)
    at run (bootstrap_node.js:425:7)
[11:51:37] Finished 'scripts' after 425 ms
[11:51:38] Finished 'hbs:compile' after 585 ms
[11:51:38] Finished 'assets' after 315 ms
/Users/lydia/Developer/sanfrancisco/node_modules/source-map/lib/source-map-generator.js:277
      throw new Error('Invalid mapping: ' + JSON.stringify({
      ^

Error: Invalid mapping: {"generated":{"line":7,"column":19223},"source":"../vendors/bootstrap/assets/stylesheets/bootstrap/mixins/_clearfix.scss","original":{"line":14,"column":-11},"name":null}
    at SourceMapGenerator_validateMapping [as _validateMapping] (/Users/lydia/Developer/sanfrancisco/node_modules/source-map/lib/source-map-generator.js:277:13)
    at SourceMapGenerator_addMapping [as addMapping] (/Users/lydia/Developer/sanfrancisco/node_modules/source-map/lib/source-map-generator.js:101:12)
    at /Users/lydia/Developer/sanfrancisco/node_modules/source-map/lib/source-map-generator.js:72:17
    at Array.forEach (native)
    at BasicSourceMapConsumer.SourceMapConsumer_eachMapping [as eachMapping] (/Users/lydia/Developer/sanfrancisco/node_modules/source-map/lib/source-map-consumer.js:155:14)
    at Function.SourceMapGenerator_fromSourceMap [as fromSourceMap] (/Users/lydia/Developer/sanfrancisco/node_modules/source-map/lib/source-map-generator.js:48:24)
    at applySourceMap (/Users/lydia/Developer/sanfrancisco/node_modules/vinyl-sourcemaps-apply/index.js:26:40)
    at filePush (/Users/lydia/Developer/sanfrancisco/node_modules/gulp-sass/index.js:105:9)
    at Object.callback (/Users/lydia/Developer/sanfrancisco/node_modules/gulp-sass/index.js:147:9)
    at options.success (/Users/lydia/Developer/sanfrancisco/node_modules/node-sass/lib/index.js:308:32)

npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/lydia/.nvm/versions/node/v7.7.1/bin/node" "/Users/lydia/.nvm/versions/node/v7.7.1/bin/npm" "start"
npm ERR! node v7.7.1
npm ERR! npm  v4.1.2
npm ERR! code ELIFECYCLE
npm ERR! NodeSchoolSF@2.0.0 start: `gulp`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the NodeSchoolSF@2.0.0 start script 'gulp'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the NodeSchoolSF package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs NodeSchoolSF
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls NodeSchoolSF
npm ERR! There is likely additional logging output above.
```

Is this a me-problem or a bug in the files as they are?

**Comment by skipzero** [comment]
oh no!! let me go back and double check it's still running on mine... where are the unit tests for the build!? 😜

i'm a mess and apparently shouldn't try coding anything anytime soon... 😕

**Comment by llkats** [comment]
@skipzero the more I think about it, the more it's feeling like we're putting the horse before the cart with these style decisions before we even have any real content.

Maybe we could close this PR for now and you could tackle some of the markup issues for now? I'll have more cycles starting soon and will dig in to those issues as well. Once we have a clearer understanding of the content we have, we can decide on the appropriate tooling. 

I'm sorry that I haven't done more leadership of this project, but I've had my hands full with Life Stuff lately. Now that that's settling down, I can hopefully be pitching in on this more frequently.

**Comment by skipzero** [comment]
I hear you on life stuff. Major shake up in my personal life and my office moved from albany, ca to SF last week. there's not a direction that something's going right, really, right now...

Sure we can do that. And it can be handled better next round... I keep getting stuck in a rebase circle with this any how. Go ahead and close and if you would suggest something you'd like me to pick up, I'd appreciate it. Happy π day, too! I'm going to spend it developing my weather station on my RPi, if I ever get to leave work.

**Comment by llkats** [comment]
Cool, thanks for understanding, @skipzero!

Closing for now.

**Comment by teone** [comment]
@skipzero @llkats sorry I've got stuck in work too... maybe we can work together on this at the next Nodeschool event! More brains more progress :)
